test/e2e/resize_nodes.go

```diff
@@ -199,6 +217,28 @@ func waitForPodsRunning(c *client.Client, pods *api.PodList) []error {
 	return e
 }
 
+func waitForPodToDisappear(c *client.Client, ns, name, podName string) error {
```

Comment:
Same comment as above.  I don't remember whether something exactly like this exists, but if it doesn't, please add this function to the general purpose e2e utility library, so that nobody else re-implements it.